### ISSUE TITLE
Benchmarks on Benchmarksuite are not displayed in order (plot/rst)

### ISSUE THREAD

**Issue by pperez** [python-recsys/benchy#4]
Benchmarks are run but the results aren't in the order they were introduced on the suite.
I think this has to do with storing the results on a dictionary ... Maybe storing them in a tuple or list would get this fixed, i'll give it a try :)

**Comment by marcelcaraciolo** [comment]
@janitux  yes i could store it as tuples, or store the order index for each benchmark and present them in order, what u think ?

**Comment by pperez** [comment]
@marcelcaraciolo i think the second option is easier to implement, on first one you'll have to change every code that threats results as a dictionary
